### ISSUE TITLE
Installation fails

### ISSUE THREAD

**Issue by jeffscottmtl** [Marginal/QLVideo#101]
Thanks for fixing .avi previews! They're working great on one of my computers, but I'm getting an "Installation failed" message when trying to install 1.96 on another. I uninstalled 1.94 (  sudo rm -rf "/Library/Application Support/QLVideo" "/Library/QuickLook/Video.qlgenerator" "/Library/Spotlight/Video.mdimporter") and now am stuck without QLVideo. I get the same error when trying to install 1.94 and 1.95. I rebooted, but the problem persists. Any suggestions will be very much appreciated. Thanks!

**Comment by Marginal** [comment]
Are you installing via Homebrew or from a release on [GitHub](https://github.com/Marginal/QLVideo/releases/latest) ?

Please try to install 1.96 again, then in a new Terminal window paste the following and press Enter:
`tail -80 /var/log/install.log`
and copy and paste the contents of the Terminal window here.

**Comment by jeffscottmtl** [comment]
Thanks very much for the reply. I've been trying via GitHub. I just downloaded a fresh copy of the pkg and tried again with the same results. I took a crack at examining the log file myself and saw this:

2022-11-16 09:06:00-05 Jeffs-CN-MacBook-Pro installd[1534]: PackageKit: Install Failed: Error Domain=PKInstallErrorDomain Code=110 "An error occurred while extracting files from the package “QLVideo_196.pkg”." UserInfo={NSLocalizedDescription=An error occurred while extracting files from the package “QLVideo_196.pkg”., NSURL=file:///Users/jeffscott/Library/CloudStorage/OneDrive-Personal/CN/QLVideo_196.pkg#QLVideo.pkg, PKInstallPackageIdentifier=uk.org.marginal.qlvideo, NSUnderlyingError=0x7ff0c0d0a560 {Error Domain=PKXARArchiveErrorDomain Code=101 "archive open failed" 

So I moved the pkg from OneDrive to iCloud Drive, tried again, and it worked! I've never had a problem installing from OneDrive before, so it may be worth looking into for others, but it's now working for me.

Thanks again!
